x-pack/legacy/plugins/rollup/public/crud_app/store/actions/detail_panel.js

```diff
@@ -10,7 +10,7 @@ import {
   CLOSE_DETAIL_PANEL,
 } from '../action_types';
 
-export const openDetailPanel = ({ panelType, jobId }) => (dispatch) => {
+export const openDetailPanel = ({ panelType, jobId, isEditMode = false }) => (dispatch) => {
```

Comment:
is `isEditMode` needed? i didn't see it being used anywhere, but i may have missed it.

Reply:
This was also something that should have been cleaned up, thanks for spotting 👍